api_review/RadialGradientBrush/RadialGradientBrush.md

```diff
@@ -0,0 +1,184 @@
+
+1. Draw a gradient using another renderer, e.g. Direct2D
+   * This would require loading additional expensive dependencies which WinUI doesn't currently rely on.
+2. Draw a custom gradient surface, e.g. using a CompositionSurfaceBrush or WriteableBitmap
+   * This would require largely reimplementing the complex CompositionRadialGradientBrush.
+3. **Rely on FallbackColor** (chosen option)
+   * Draw a solid color, as specified by the [XamlCompositionBrushBase.FallbackColor](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Media.XamlCompositionBrushBase.FallbackColor) (which RadialGradientBrush will inherit from its base class).
+
+Using FallbackColor follows the precedence of [AcrylicBrush](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Media.AcrylicBrush). And since CompositionRadialGradientBrush should be available downlevel with WinUI 3, this seems like a reasonable approach in the meantime.
+
+## InterpolationSpace
+
+Note that the underlying [CompositionRadialGradientBrush](https://docs.microsoft.com/uwp/api/windows.ui.composition.compositionradialgradientbrush) currently only supports the following [CompositionColorSpace](https://docs.microsoft.com/uwp/api/windows.ui.composition.compositioncolorspace) values:
+
+* `Auto`
+* `Rgb`
+* `RgbLinear`
+
+Applying any other interpolation color space will have no effect.
+
+# Description
+
+RadialGradientBrush paints an area with a radial gradient. A center point defines the origin of the gradient, and an ellipse defines the outer bounds of the gradient.
+
+# Examples
+
+This example creates a radial gradient with six gradient stops and uses it to paint a Rectangle.
+
+![A rectangle filled with a radial gradient.](images/ColorRadialGradientBrush.png)
+
+XAML
+```XAML
+<Page
+  xmlns:media="using:Microsoft.UI.Xaml.Media">
+
+  <Rectangle Width="200" Height="200">
+      <Rectangle.Fill>
+          <media:RadialGradientBrush>
+              <GradientStop Color="Blue" Offset="0.0" />
+              <GradientStop Color="Yellow" Offset="0.2" />
+              <GradientStop Color="LimeGreen" Offset="0.4" />
+              <GradientStop Color="LightBlue" Offset="0.6" />
+              <GradientStop Color="Blue" Offset="0.8" />
+              <GradientStop Color="LightGray" Offset="1" />
+          </media:RadialGradientBrush>
+      </Rectangle.Fill>
+  </Rectangle>
+
+</Page>
+```
+
+This example creates a radial gradient that uses Absolute mapping mode with custom values for `EllipseCenter`, `EllipseRadius` and `GradientOrigin`:
+
+![A rectangle filled with an offset radial gradient.](images/OffsetRadialGradientBrush.png)
+
+```XAML
+<Page
+  xmlns:media="using:Microsoft.UI.Xaml.Media">
+
+  <Rectangle Width="200" Height="200">
+      <Rectangle.Fill>
+          <media:RadialGradientBrush
+            MappingMode="Absolute"
+            EllipseCenter="0.5,0.5"
```

Comment:
This is not accurate. When in absolute mapping mode, In order to get results that match the picture above, the values will need to be 
EllipseCenter="50,50"
EllipseRadius="100,100"
GradientOrigin="50,0"

Reply:
I got confused about mapping modes during the conversion, I've updated it. But shouldn't the GradientOrigin for this picture be 100,50?